### PR TITLE
Add debug logging to telnet commands for minicircuits attenuator

### PR DESCRIPTION
**Pull request by xianyuanjia**
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/772)

### CODE REVIEW

File: mobly/controllers/attenuator_lib/telnet_scpi_client.py [hunk 1]
```diff
@@ -72,6 +73,10 @@ def cmd(self, cmd_str, wait_ret=True):
     match_idx, match_val, ret_text = self._tn.expect(
         [_ascii_string("\S+" + self.rx_cmd_separator)], 1)
 
+    logging.debug('Telnet Command: {}'.format(cmd_str))
```
Comment (xpconanfan) [inline mobly/controllers/attenuator_lib/telnet_scpi_client.py:76]: use logging's own string formatting instead

**Review by xpconanfan — CHANGES_REQUESTED** [review]
Revert the change and add logging in separate PR

File: mobly/controllers/attenuator_lib/telnet_scpi_client.py [hunk 2]
```diff
@@ -72,6 +73,9 @@ def cmd(self, cmd_str, wait_ret=True):
     match_idx, match_val, ret_text = self._tn.expect(
         [_ascii_string("\S+" + self.rx_cmd_separator)], 1)
 
+    logging.debug('Telnet Command: %s', cmd_str)
```
Comment (xpconanfan) [inline mobly/controllers/attenuator_lib/telnet_scpi_client.py:76]: let's log these info in one line to be consistent with the other cmd logging patterns in Mobly, like `adb`

**Review by xpconanfan — CHANGES_REQUESTED** [review]

**Review by xianyuanjia — COMMENTED** [review]
Done.

> *[Reviewable](https://reviewable.io/reviews/google/mobly/772#-:-MpEen8r9elf1vMaOS8C:b-896fix)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @xianyuanjia and @xpconanfan)

___
*[mobly/controllers/attenuator_lib/telnet_scpi_client.py, line 76 at r1](https://reviewable.io/reviews/google/mobly/772#-MotzdsBUa5ZL0To77xA-r1-76:-MpEej2IDbOeu_SMtXW0:b-896fix) ([raw file](https://github.com/google/mobly/blob/9dfb013d7c1e35d6475229e8714f2399d6b99acc/mobly/controllers/attenuator_lib/telnet_scpi_client.py#L76)):*
<details><summary><i>Previously, xpconanfan (Ang Li) wrote…</i></summary><blockquote>

use logging's own string formatting instead
</blockquote></details>

Done.
___
*[mobly/controllers/attenuator_lib/telnet_scpi_client.py, line 76 at r2](https://reviewable.io/reviews/google/mobly/772#-MotzdsBUa5ZL0To77xA-r2-76:-MpEeieg6bDlEqOHFbHd:b-896fix) ([raw file](https://github.com/google/mobly/blob/9f3b97292f92be5cbd8d52a3ab95084c985dec6a/mobly/controllers/attenuator_lib/telnet_scpi_client.py#L76)):*
<details><summary><i>Previously, xpconanfan (Ang Li) wrote…</i></summary><blockquote>

let's log these info in one line to be consistent with the other cmd logging patterns in Mobly, like `adb`
</blockquote></details>

Done.

**Comment by xpconanfan** [comment]
We don't need this any more?

**Comment by xianyuanjia** [comment]
It is no longer needed. Closing.

**Pull request closed without merging**